docs/getting-started/configuration.md

```diff
@@ -90,10 +90,15 @@ backend:
       user: ${POSTGRES_USER}
       password: ${POSTGRES_PASSWORD}
       # https://node-postgres.com/features/ssl
-      # ssl: require # see https://www.postgresql.org/docs/current/libpq-ssl.html Table 33.1. SSL Mode Descriptions (e.g. require)
-        #ca: # if you have a CA file and want to verify it you can uncomment this section
-        #$file: <file-path>/ca/server.crt
-        # highlight-add-end
+      # ssl:
+      #   host: ${POSTGRES_HOST}
```

Comment:
host shouldn't be in here right?

Reply:
I got this from [this issue](https://github.com/brianc/node-postgres-docs/issues/79).
In my case with CloudSQL it was needed as otherwise the certificate verification failed, but I can add a comment and link the issue!